lms/static/js/discussions_management/views/divided_discussions_inline.js

```diff
@@ -13,17 +13,18 @@
                     },
 
                     initialize: function(options) {
-                        this.template = HtmlUtils.template($('#cohort-discussions-inline-tpl').text());
-                        this.cohortSettings = options.cohortSettings;
+                        this.template = HtmlUtils.template($('#divided-discussions-inline-tpl').text());
+                        this.discussionSettings = options.discussionSettings;
                     },
 
                     render: function() {
-                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions'),
-                            inline_discussions = this.model.get('inline_discussions');
+                        var alwaysDivideInlineDiscussions = this.discussionSettings.get(
```

Comment:
> How about you switch inlineDiscussions to first so you don't need to break this.discussionSettings?

This does not solve the issue because the eslint standards require an indent.

I have updated it to look more like line 124.